src/learning-header/LearningHeader.jsx

```diff
@@ -28,9 +29,16 @@ LinkedLogo.propTypes = {
 };
 
 function LearningHeader({
-  courseOrg, courseNumber, courseTitle, intl, showUserDropdown,
+  courseOrg, courseTitle, intl, showUserDropdown,
 }) {
   const { authenticatedUser } = useContext(AppContext);
+  const [logoOrg, setLogoOrg] = useState(null);
+
+  useEffect(() => {
+    if (courseOrg) {
+      getCourseLogoOrg(courseOrg).then((logoOrgUrl) => { setLogoOrg(logoOrgUrl); });
```

Comment:
@dcoa Can you help me understand why you're checking if courseOrg exists as a prop before getting the logo from the api function `getCourseLogoOrg`. Also, can you explain the courseOrg as an argument for the function, taking in consideration that you're fetching the route url via the `window` global? Everything here looks good. This is just a small clarification request. Thanks in advance.

Reply:
The reason to validate if courseOrg existed is to avoid calling the API if the course is not displayed.

Using courseOrg as an argument is a mistake because the function doesn't have parameters, I will fix that, thanks for noticing.